tools/xtce_generation/telemetry_generator.py

```diff
@@ -298,4 +298,4 @@ def generate_telemetry_xml(self, output_xml_path, container_name):
         Et.indent(tree, space="\t", level=0)
 
         # Use the provided output_xml_path
-        tree.write(output_xml_path, encoding="utf-8", xml_declaration=True)
+        tree.write(output_xml_path, encoding="UTF-8", xml_declaration=True)
```

Comment:
Did this need to be changed? The standard library examples use lowercase.
https://docs.python.org/3/howto/unicode.html#reading-and-writing-unicode-data

Reply:
oh ok. I will change it back to lowecase. I changed it thinking it may be causing issue but in reality it was the IntegerParameterType that was causing issue.